### DISCUSSION TITLE
Préciser le pourcentage de durable dont on parle?

### DISCUSSION THREAD

**Discussion by ghost** [betagouv/ma-cantine#1784] [Idées]
Une utilisatrice m'a fait remarquer qu'elle ne savait pas si le pourcentage de "produits durables et de qualité" qui s'affiche à la fois sur la page "Nos Cantines", la page "Statistiques" et la fiche d'une cantine désignait le pourcentage du **total de produits durables et de qualité** ou celui des **autres produits durables et de qualité**. Elle m'a confié que sa confusion était due à la formulation de la loi, qui ne dit pas "30% de durables **et** 20% de bio", mais "50% de durables **dont** 20% de bio". Autant sa confusion n'a pas lieu d'être dès lors que le pourcentage de durable est inférieur à celui de bio, autant c'est compréhensible quand  c'est l'inverse.

Je suggère donc de préciser à chaque fois "(hors bio)" à la fin de "produits durables et de qualité".

Votre avis, @JenniferStephan, @hfroot, @alemangui, @valeriemerle ?
Vous avez déjà eu des commentaires similaires, @gauthierandre, @Greta712, Pauline ?

<img width="753" alt="Capture d’écran 2022-08-16 à 16 18 59" src="https://user-images.githubusercontent.com/93709421/184910452-92963a99-a910-406f-a64e-de08b6dddaa8.png">
<img width="1062" alt="Capture d’écran 2022-08-16 à 16 19 07" src="https://user-images.githubusercontent.com/93709421/184910460-061fa322-769e-4194-8616-c6744b99625c.png">

**Comment by alemangui** [comment]
D'accord pour ajouter "hors bio", ça rajoute de la clarté

**Reply by valeriemerle** [reply]
Bonne idée. 
Meme remarque dejà entendue de mon coté. Il faut aussi faire le changement sur l'affiche convives et les stat régionales

**Reply by alemangui** [reply]
Carte ajoutée dans le projet :+1:

**Reply by ghost** [reply]
Nice
